src/modules/balance/components/TokenDeleteCustom.vue

```diff
@@ -146,15 +185,66 @@ export default {
         : `Are you sure you want to remove ${
             this.selectedTokens.length > 1 ? 'these tokens' : 'this token'
           }?`;
+    },
+    displaySelectedTokens() {
+      return this.selectedTokens;
+    },
+    loading() {
+      return this.loadingWalletInfo;
+    },
+    preselectedTokens() {
+      const tokens = [];
+      for (const token of this.formattedHiddenTokens) {
+        const newObj = this.formattedAllTokens.filter(item => {
+          return item.address === token.address;
+        })[0];
+        tokens.push(newObj);
+      }
+      return tokens;
+    }
+  },
+  watch: {
+    loading(val) {
+      if (!val && !this.preselected) {
+        this.selectedTokens = this.preselectedTokens;
+        this.preselected = true;
+      }
+    },
+    open(val) {
+      if (val && !this.preselected && !this.loading) {
+        this.selectedTokens = this.preselectedTokens;
+        this.preselected = true;
+      }
     }
   },
   methods: {
-    ...mapActions('custom', ['deleteAll', 'deleteToken']),
+    ...mapActions('custom', [
+      'deleteAll',
+      'deleteToken',
+      'setHiddenToken',
+      'deleteHiddenToken'
+    ]),
+    // preselectTokens() {
```

Comment:
remove commented code

Reply:
The file is no longer used as I replaced it with the edit token form so I will delete it